lib/Config.js

```diff
@@ -28,7 +28,15 @@ var get = function (key) {
     return config[key];
 };
 
+// twilio-notifications
+var twilio_notifications_host = process.env.TWILIO_NOTIFICATIONS_HOST;
+var twilio_notifications_port = process.env.TWILIO_NOTIFICATIONS_PORT || 3007;
+exports.twilio_notifications_host = twilio_notifications_host;
+exports.twilio_notifications_port = twilio_notifications_port;
```

Comment:
normally I would not bake config and env variable setup into a lib like this .  it makes it unclear where the dependency lies in the top level service.  

IF config was needed in a lib (or lib like thing)  I would prefer its passed in as a parameter when initating the lib in the calling code.

but an exception can be made here because of the unique circumstances of this code base, and what calls it as a dependency.

but wanted to express that opinion here